docs/vNext/tutorials_fabric_existing.md

```diff
@@ -0,0 +1,629 @@
+The network configuration file is the file required by Caliper workers to create a connection to the existing Fabric network so that they may submit transactions. It resembles the Fabric common connection profile, augmented with additional required fields. The file can be in YAML or JSON format, this tutorial shows the JSON format. 
+
+### Create a Template Network Configuration File
+Under the **networks** folder create a template file called **networkConfig.json** with the following content:
+``` json
+{
+    "version" : "1.0",
+    "name": "Caliper test",
+    "caliper" : {
+        "blockchain": "fabric"
+    },
+    "clients" : {
+    },
+    "channels" : {
+    },
+    "organizations" : {
+    },
+    "peers" : {
+    }
+}
+```
+__version__: The version of the configuration file being used. Only "1.0" is accepted.
+
+__name__: The name for the configuration, in this instance "Caliper test".
+
+__caliper__: Indicates to Caliper the SUT that is being targeted, and may contain additional start/end commands that are not required within this tutorial. For the purposes of this tutorial, we are targeting a `fabric` network.
+
+__clients__: Lists the identities to be used within the performance test
+
+__channels__: Describes the Hyperledger Fabric channels available, their state, and the smart contracts deployed on these channels
+
+__organizations__: The Hyperledger Fabric organizations that are to be used within the benchmarking
+
+__peers__: The Hyperledger Fabric peers that are to be used within the benchmarking
+
+### Populating The Template File
+Following the test-network tutorial, a Common Connection Profile (CCP) is generated; we will use this file to help populate the Hyperledger Fabric specific elements of the Caliper network configuration file. The template will have some unique certificates for the Hyperledger Fabric network that was created, therefore you will notice differences to contents shown in subsequent stages; it is essential that you use the certificates created for interaction with the test-network you wish to target.
+
+We will be using Org1 to connect in this example. To find the generated JSON or YAML CCP files, look in **fabric-samples** -> **test-network** -> **organizations** -> **peerOrganizations** -> **org1.example.com**. We assume use of the json CCP **connection-org1.json**. 
+
+#### Organizations and Peers
+These are populated using the contents of the CCP:
+1. Copy the `organizations` and `peers` elements of the CCP into the Caliper network configuration file. 
+2. Within the `organizations.Org1` object, delete the `certificateAuthorities` listing. This is done for two reasons:
+    
+    i) We will not be using the certificate authority to register or enrol
+
+    ii) There has been a breaking change delivered to the CCP that is incompatible with the validator Caliper applies to the configuration files.
+
+Your Caliper network configuration file should now look like this:
```

Comment:
should noew look `similar` to this as the <UNIQUE CONTENT> stanza may not be obvious to some